src/components/ColorPalette/ColorPalette.jsx

```diff
@@ -0,0 +1,24 @@
+import "./ColorPalette.css"
+import { useState } from "react"
+import { useNote } from "../../contexts"
+export const ColorPalette = ({ activeElement, setActiveElement }) => {
+    const { setNoteData } = useNote()
+
+    const paletteColors = ['#FFFFFF', '#FCFFA6', '#cbf0f8', '#aecbfa', '#ccff90', '#CEE5D0', '#fdcfe8', '#F0D9FF']
```

Comment:
Here, if you dont want specific colors, you can also generate random colors here